client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -147,43 +145,64 @@ export const PrescriptionLineEditForm: React.FC<
   // See https://github.com/msupply-foundation/open-msupply/issues/2727
   // and https://github.com/msupply-foundation/open-msupply/issues/3532
   const debouncedAllocate = useDebouncedValueCallback(
-    (quantity, packSize) => {
-      allocate(quantity, packSize);
+    (numPacks, packSize) => {
+      allocate(numPacks, packSize);
     },
     [],
     500,
     [draftPrescriptionLines] // this is needed to prevent a captured enclosure of onChangeQuantity
   );
 
-  const handleIssueQuantityChange = (inputQuantity?: number) => {
-    // this method is also called onBlur... check that there actually has been a change
-    // in quantity (to prevent triggering auto allocation if only focus has moved)
-    if (inputQuantity === issueQuantity) return;
+  const handleIssueQuantityChange = (inputUnitQuantity?: number) => {
+    // this method is also called onBlur... check that there actually has been a
+    // change in quantity (to prevent triggering auto allocation if only focus
+    // has moved)
+    if (inputUnitQuantity === issueUnitQuantity) return;
+
+    const quantity = inputUnitQuantity === undefined ? 0 : inputUnitQuantity;
+    setIssueUnitQuantity(quantity);
+
+    const packSize =
```

Comment:
Won't this always be 1 now that we're issuing/allocating in units?
I'm ok with keeping this logic in case we want to re-integrate the packsize selection in future. Maybe that was your thinking?

Reply:
Yes, probably should be just 1. I was just finding that every time I changed something like this I broke something else in an unpredictable way, so was trying to minimize changes. It's still doing some things I don't fully understand, so pulling on as few threads as I can for now.